src/Features/Core/Portable/Completion/Providers/AbstractSymbolCompletionProvider.cs

```diff
@@ -128,7 +141,7 @@ private bool ShouldIncludeInTargetTypedCompletionList(ISymbol symbol, ImmutableA
                     foreach (var symbol in symbolGroup)
```

Comment:
Can this loop be parallelized? If so, would that help?

Reply:
Maybe? I'm not sure it's worth that investment yet.